packages/telemetry/src/slog-to-otel.js

```diff
@@ -571,52 +788,44 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         }
         break;
       }
-      case 'bundle-kernel-start': {
-        spans.push(`bundle-kernel`);
-        break;
-      }
-      case 'bundle-kernel-finish': {
-        spans.pop('bundle-kernel');
+      case 'kernel-stats': {
+        spans.top()?.addEvent('kernel-stats', cleanAttrs(slogAttrs.stats), now);
```

Comment:
I had missed this originally, we're ok with the stats being flat under the `agoric.` prefix, and not nested under a e.g. `agoric.stats.` prefix?

Reply:
I'm OK with it.  Frankly, I find the whole name-spacing of attribute names thing kind of dumb and useless anyway.

Reply:
Yeah I'm just paranoid of conflicts/overloads in otel sinks which may assume the attribute name to have some kind of semantic meaning if they appear on differently named spans.